__tests__/app.test.js

```diff
@@ -31,15 +32,33 @@ describe ( "app" , () => {
 			}) ;
     }) ;
   })
+
+  describe ( "GET /api" , () => {
+
+    test( "200: should return a JSON of all available endpoints" , () => {
+      
+      return request( app )
+      .get( "/api" )
+      .expect( 200 )
+      .then( ( { body } ) => {
+        return fs.promises.readFile( `${__dirname}/../endpoints.json` , "utf8" )
+        .then( ( endpoints ) => {
+          expect( body ).toEqual( JSON.parse( endpoints ) ) ;
+        }) ;
```

Comment:
This works but is a little over complicated. Dont forget that you can directly require in .json files and access the objects within without having to using the fs library.